codec_specs.md

```diff
@@ -263,6 +263,16 @@ Description: FFV1 is a lossless intra-frame video encoding format designed to ef
 
 Initialisation: For FFV1 versions 0 or 1, `Private Data` SHOULD NOT be written. For FFV1 version 3 or greater, the `Private Data` MUST contain the FFV1 Configuration Record structure, as defined in https://tools.ietf.org/html/draft-niedermayer-cellar-ffv1-01#section-4.1, and no other data.
 
+### V_AVS2_P2
+
+Codec ID: V_AVS2
```

Comment:
For the other codecs list here the Codec ID and the heading are always identical. Therefore I don't understand what that `_P2` suffix is supposed to mean.

Reply:
The P2 and P3 suffix were added to distinguish two different part of AVS2 standard (the video part and audio part). Well, it's looks like there is no need to do that. I will modify and re-submit them soon.